### PR TITLE
added PHP 7.3.0

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

**Comment by DracoBlue** [comment]
Updated to RC4

File: 7.3/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,99 @@
+FROM php:7.3RC01-fpm
```
Comment (OskarStark) [inline 7.3/Dockerfile]: This should be changed too

**Comment by OskarStark** [comment]
And of course a rebase should be done

File: 7.3/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+# ssh2
+RUN apt-get update && apt-get install -y libssh2-1-dev libssh2-1
```
Comment (DracoBlue) [inline 7.3/Dockerfile]: Travis says:

```
The command '/bin/sh -c pecl install ssh2-1.1.2' returned a non-zero code: 1
```

File: 7.3/Dockerfile (hunk 1 above)
Comment (DracoBlue) [reply 7.3/Dockerfile]: But it does not matter when building specific versions

File: 7.3/Dockerfile (hunk 2 above)
Comment (DracoBlue) [reply 7.3/Dockerfile]: @OskarStark

**Comment by DracoBlue** [comment]
Travis says:

```
The command '/bin/sh -c pecl install ssh2-1.1.2' returned a non-zero code: 1
```

@OskarStark

**Comment by OskarStark** [comment]
Strange, this is the last release which should work with PHP 7.0 and above

https://pecl.php.net/package/ssh2

**Comment by mbehrsing** [comment]
shouldn't this PR be splitted? this update to php 7.3 also contains integration of ssh2 + unzip:

```
diff --git a/7.2/Dockerfile b/7.3/version-Dockerfile
index 870740c..7c733d0 100644
--- a/7.2/Dockerfile
+++ b/7.3/version-Dockerfile
@@ -1,4 +1,10 @@
-FROM php:7.2-fpm
+FROM php:7.3-fpm
+
+# ssh2
+RUN apt-get update && apt-get install -y libssh2-1-dev libssh2-1
+RUN pecl install ssh2-1.1.2
+RUN echo extension=ssh2.so >> /usr/local/etc/php/conf.d/ssh2.ini
+# ssh2

 RUN apt-get update && apt-get install -y libz-dev libmemcached-dev
 RUN pecl install memcached-3.0.3
@@ -37,7 +43,7 @@ RUN apt-get update && apt-get install libldap2-dev -y && \
     rm -rf /var/lib/apt/lists/* && \
     docker-php-ext-configure ldap --with-libdir=lib/x86_64-linux-gnu/ && \
     docker-php-ext-install ldap
-# ldap
+# ldap

 RUN docker-php-ext-configure pcntl --enable-pcntl \
     && docker-php-ext-install pcntl
@@ -48,9 +54,9 @@ RUN echo "zend_extension=`php -i | grep ^extension_dir | cut -f 3 -d ' '`/xdebug
 RUN docker-php-ext-install soap

 ENV COMPOSER_HOME=/composer
-COPY --from=composer:1.6.5 /usr/bin/composer /usr/local/bin/composer
+COPY --from=composer:1.7.1 /usr/bin/composer /usr/local/bin/composer

-RUN apt-get update && apt-get install -y git vim mysql-client rsync sshpass bzip2 msmtp
+RUN apt-get update && apt-get install -y git vim mysql-client rsync sshpass bzip2 msmtp unzip

 ADD php.ini /usr/local/etc/php/php.ini

@@ -73,7 +79,7 @@ ENV PHP_DATE_TIMEZONE="" \
     PHP_MAX_INPUT_VARS=1000 \
     PHP_MEMORY_LIMIT=128M \
     PHP_POST_MAX_SIZE=8M \
-    PHP_SENDMAIL_PATH="/usr/sbin/sendmail -t -i" \
+    PHP_SENDMAIL_PATH="/usr/sbin/sendmail -t -i" \
     PHP_SESSION_SAVE_HANDLER=files \
     PHP_SESSION_SAVE_PATH="" \
     PHP_UPLOAD_MAX_FILESIZE=2M \
@@ -90,4 +96,4 @@ WORKDIR /usr/src/app
 # imagick
 RUN apt-get update && apt-get install -y libmagickwand-dev \
     && pecl install imagick \
-    && docker-php-ext-enable imagick
+    && docker-php-ext-enable imagick
\ No newline at end of file
```

**Comment by DracoBlue** [comment]
@mbehrsing your local version of master is outdated. ssh2 and unzip should be in all php versions

**Comment by mbehrsing** [comment]
just noticed..my mistake

**Comment by DracoBlue** [comment]
The memcached extension does not yet support PHP 7.3. :(

**Comment by DracoBlue** [comment]
A patch for memcached:

https://github.com/websupport-sk/pecl-memcache/pull/30

**Comment by DracoBlue** [comment]
And official: https://github.com/php-memcached-dev/php-memcached/issues/408

**Comment by DracoBlue** [comment]
Updated to RC6

**Comment by OskarStark** [comment]
I cleanup the commits, later we can revert the ssh2 commit

**Pull request merged**